docs/sections/components/step/global_step.md

```diff
@@ -0,0 +1,107 @@
+# GlobalStep
+
+The [`GlobalStep`][distilabel.steps.GlobalStep] is a subclass of [`Step`][distilabel.steps.Step] that is used to define a step that requires the previous steps to be completed to run, since it will wait until all the input batches are received before running. This step is useful when you need to run a step that requires all the input data to be processed before running.
+
+## Working with GlobalSteps
+
+The [`GlobalStep`][distilabel.steps.GlobalStep] is intended to be used within the scope of a [`Pipeline`][distilabel.pipeline.Pipeline] and after some previous steps have been defined. But it can also be used as a standalone [`Step`][distilabel.steps.Step] if needed, but if that's the case, then using [`Step`][distilabel.steps.Step] would be more appropriate.
```

Comment:
```suggestion
The [`GlobalStep`][distilabel.steps.GlobalStep] is intended to be used within the scope of a [`Pipeline`][distilabel.pipeline.Pipeline] and after some previous steps have been defined. But it can also be used as a standalone [`Step`][distilabel.steps.Step] if needed. If that's the case, then using [`Step`][distilabel.steps.Step] would be more appropriate.
```

Reply:
```suggestion
The [`GlobalStep`][distilabel.steps.GlobalStep] is intended to be used within the scope of a [`Pipeline`][distilabel.pipeline.Pipeline] and after some previous steps have been defined. Alternatively, it can also be used as a standalone [`Step`][distilabel.steps.Step] if needed, but if that's the case, then using [`Step`][distilabel.steps.Step] would be more appropriate.
```